File: inf1000/uke1/Kroppstemperatur.java
/**
1.10: Kroppstemperatur
*/

import java.util.Scanner;
class Kroppstemperatur {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        double minTemp = 36.5;
        double maksTemp = 37.5;

        System.out.print("Skriv inn din Kroppstemperatur: ");
        double innTemp = Double.parseDouble(in.next());

        if (innTemp < minTemp || innTemp > maksTemp) {
            System.out.println("Du har feber!");
        } else {
            System.out.println("Du er frisk!");
        }
    }
}

File: in1000/host2020/uke2/kroppstemperatur.py
# 02.08: kroppstemperatur

temp_L = 36.5
temp_H = 37.5

print("| Har du feber? |")
temperatur = float(input("Har du feber? Skriv inn din kroppstemperatur: "))

if (temperatur < temp_L):
    print("Din kroppstemperatur er under normal kroppstemperatur.")
elif (temperatur > temp_H):
    print("Din kroppstemperatur er over normal kroppstemperatur.")
else:
    print("Din kroppstemperatur er normal.")
